requirements.txt

```diff
@@ -1,8 +1,9 @@
 torch>=1.6
 torchmetrics>=0.2.0
-torchvision
 pytorch-lightning>=1.2.0
+torchvision
+kornia>=0.4
```

Comment:
```suggestion
git+https://github.com/kornia/kornia
```

Reply:
this does not add KOrnian in Gh dependencies..